themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -28,15 +28,28 @@
       <? foreach (($top = $results->getRecommendations('top')) as $current): ?>
         <?=$this->recommend($current)?>
       <? endforeach; ?>
-      <?=$this->transEsc("Showing")?>
-      <strong><?=$this->localizedNumber($results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($results->getEndRecord())?></strong>
-      <? if (!isset($view->skipTotalCount)): ?>
-        <?=$this->transEsc('of')?> <strong><?=$this->localizedNumber($recordTotal)?></strong>
+      <?
+        $transParams = [
```

Comment:
Can you check EDS specifically for me? I had no problem with Solr and Summon.
![image](https://cloud.githubusercontent.com/assets/451107/23083662/d0c42dc4-f52c-11e6-9b4d-7869c60d6f0e.png)